server/container/mailbox-adapter/src/main/java/org/apache/james/adapter/mailbox/MailReprocessingManagementMBean.java

```diff
@@ -0,0 +1,7 @@
+package org.apache.james.adapter.mailbox;
+
+public interface MailReprocessingManagementMBean {
+    void reprocessAllMails(String repositoryPath);
+    void reprocessMail(String repositoryPath, String emailKey);
+    String[] listMailRepository() throws Exception;
```

Comment:
Specifying an array as part of an API seems strange to me. 

Working with a Collection or a List seems nicer.